rc/lsp.kak

```diff
@@ -1226,6 +1226,10 @@ define-command -hidden lsp-inlay-hints -docstring "lsp-inlay-hints: request inla
     }
 }
 
+define-command lsp-inlay-hint-apply-textedit -docstring "Apply the textedit for the inlay hint nearest the cursor" %{
+    lsp-send kakoune/inlay-hint-apply-textedit %val{selection_desc}
```

Comment:
if we want to handle all selections, we'd want to pass `selections_desc'.

But lik esome other comments, feel free to leave it for later if you don't want to spend the extra work now.